internal/web_server/web/handlers/api/admin_api.go

```diff
@@ -843,3 +848,57 @@ func (aah *AdminApiHandler) randomizeAlternatives(c echo.Context) error {
 	return utils.Render(c, http.StatusOK, dashboard_components.QuestionAlternativesInput(alternatives))
 
 }
+
+// Add the given word to the username tables.
+func (aah *AdminApiHandler) addUsername(c echo.Context) error {
+	word := c.FormValue("usernameWord")
+	table := c.QueryParam("tableId")
```

Comment:
Use kebab case for all input names and query parameters

Reply:
doje